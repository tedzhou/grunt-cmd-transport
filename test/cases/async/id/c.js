/**
 * @fileoverview:
 * @author: tedzhou
 * @date: 13-9-20 下午6:22
 */
define(function(require){
	require('id/d');
	require('id/e');
})