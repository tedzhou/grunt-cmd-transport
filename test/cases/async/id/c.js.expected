/**
 * @fileoverview:
 * @author: tedzhou
 * @date: 13-9-20 下午6:22
 */
define("id/c", [ "id/d", "id/e" ], function(require) {
    require("id/d");
    require("id/e");
});